tools/pubsys/src/main.rs

```diff
@@ -182,7 +182,7 @@ mod error {
         #[snafu(display("Logger setup error: {}", source))]
         Logger { source: log::SetLoggerError },
 
-        #[snafu(display("Failed to publish AMI: {}", source))]
+        #[snafu(display("Error during publish-ami command: {}", source))]
```

Comment:
nit: I would actually keep this one as it was. We call this using the `publish-ami` command, but pubsys can be called on its own, so the original statement was more accurate and doesn't change any meaning.

Reply:
I think the original statement was problematic because I think `"Failed to publish AMI: {}"` made it sound like a single AMI failed. How about `"Failure during AMI publicaction: {}"`?

Reply:
> nit: I would actually keep this one as it was. We call this using the `publish-ami` command, but pubsys can be called on its own, so the original statement was more accurate and doesn't change any meaning.

Actually, I'm not sure what you mean by "pubsys can be called on its own". This error variant is only used in one place, which is during the `publish-ami` command. And this is a main.rs file so there's no way for it to be used somewhere else. Can you clarify the use-case where you think this error message would be incorrect?

Reply:
Sorry, I was just thinking of `cargo make publish-ami`. We're good now.